BREAKING_CHANGES.md

```diff
@@ -1,3 +1,11 @@
+2023-03-22 2624af6
+
+disk config now needs to be inside a disko.devices attrset always
```

Comment:
It's not clear from this one line what the changes would need to be fore a user.
Also the tutorial needs an update: https://github.com/nix-community/disko/blob/master/docs/quickstart.md

Reply:
https://github.com/nix-community/disko/pull/162/files#diff-988a5392cf907f7c5728b8ac90445facc7f835f3c00375764ec99769dcde2d5a

also people usually don't need to change anything if they haven't used the cli for installation. So it's unclear what kind of structure people have used before